doc/modules/cassandra/partials/table-properties.adoc

```diff
@@ -105,6 +105,22 @@ If your application uses batch operations, consider the possibility that decreas
 The configuration/cass_yaml_file.html#batchlog_replay_throttle[batchlog_replay_throttle] property in the cassandra.yaml file give some control of the batch replay process.
 The most important factors, however, are the size and scope of the batches you use.
 
+*memtable*::
+Configures the memtable for the table.
+Four choices are:
```

Comment:
The choices here are the ones defined in `cassandra.yaml`, in `memtable.configurations`. With CASSANDRA-18753 (which should be merged soon), `trie` and `skiplist` will be defined in the supplied `cassandra.yaml` and `cassandra-latest.yaml`.

Reply:
OK, didn't realize that was the case. I just looked at what I found in the resources, like the MD file.

Reply:
Changed the wording here to match CASSANDRA-18753.